File: app/src/main/java/com/vutrankien/t9vietnamese/android/T9Keyboard.java
package com.vutrankien.t9vietnamese.android;

import android.content.Context;
import android.inputmethodservice.Keyboard;

/**
 * Created by vutrankien on 17/07/24.
 */

class T9Keyboard extends Keyboard {
    T9Keyboard(Context context, int xmlLayoutResId) {
        super(context, xmlLayoutResId);
    }

}

File: app/src/main/java/com/vutrankien/t9vietnamese/android/T9Keyboard.kt
package com.vutrankien.t9vietnamese.android

import android.content.Context
import android.inputmethodservice.Keyboard

/**
 * Created by vutrankien on 17/07/24.
 */
internal class T9Keyboard @JvmOverloads constructor(context: Context?, xmlLayoutResId: Int = R.xml.t9) : Keyboard(context, xmlLayoutResId)